en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -6,190 +6,159 @@ iconBrand: php
 
-## PSR-2
+## General
 
-Our goal is [PSR-2 Coding Standards](https://www.php-fig.org/psr/psr-2/) compliance.
-Since this affects existing APIs, some details like method casing will be iterated on in the next releases.
-For example, many static methods will need to be changed from lower underscore to lower camel casing. 
- 
-## Spelling
+- Aim for compliance with the [PSR-12 coding standards](https://www.php-fig.org/psr/psr-12/).
+- Prioritise the human readability of code over "clever" code that reduces the number of lines at the expense of readability.
+- All symbols and documentation should use UK-English spelling, for example "behaviour" instead of "behavior" except when necessitated by third party conventions, for example using PHP's `Serializable` interface.
+- Avoid writing HTML in PHP code. Instead, put HTML in template files.
 
-All symbols and documentation should use UK-English spelling (e.g. "behaviour" instead of "behavior"),
-except when necessitated by third party conventions (e.g using PHP's `Serializable` interface).
-
-## Configuration Variables
-
-Silverstripe CMS's [Config API](/developer_guides/configuration/configuration/) can read its defaults from variables declared as `private static` on classes.
-As opposed to other variables, these should be declared as lower case with underscores.
-
-
-```php
-class MyClass
-{
-    private static $my_config_variable = 'foo';
-}
-```
-
-## Prefer identical (===) comparisons over equality (==)
+## Naming conventions
 
-Where possible, use type-strict identical comparisons instead of loosely typed equality comparisons.
-Read more in the PHP documentation for [comparison operators](https://php.net/manual/en/language.operators.comparison.php) and [object comparison](https://php.net/manual/en/language.oop5.object-comparison.php).
-
-
-```php
-// good - only need to cast to (int) if $a might not already be an int
-if ((int)$a === 100) {
-    doThis();
-}
-
-// bad
-if ($a == 100) {
-    doThis();
-}
-```
-
-## Separation of Logic and Presentation
-
-Try to avoid using PHP's ability to mix HTML into the code.
-
-
-```php
-// PHP code
-public function getTitle() 
-{
-    return "<h2>Bad Example</h2>";
-}
-```
+### Casing
 
-```ss
-// Template code
-$Title
-```
+- Follow [PSR-1](https://www.php-fig.org/psr/psr-1/#1-overview) for naming class names, method names, and class constants
+- Use [snake_case](https://en.wikipedia.org/wiki/Snake_case) for configuration properties (i.e. any `private static` properties not annotated with `@internal`)
+- Use [camelCase](https://en.wikipedia.org/wiki/Camel_case) for all other class properties
 
-Better: Keep HTML in template files:
+### Accessors
 
+- Prefix with "get" for getter methods and 'set' for setter methods, for example `getValue()` and `setValue()`.
```

Comment:
Whe're using double quotes for the first one and single quotes for the second one.

```suggestion
- Prefix with `get` for getter methods and `set` for setter methods, for example `getValue()` and `setValue()`.
```

Reply:
Updated